.github/workflows/binary-applications.yml

```diff
@@ -176,10 +176,10 @@ jobs:
                   path: ~/opt/anaconda3
                   key: mac-py${{ env.PYTHON_VERSION }}-${{ env.PYTHON_ARCH}}-conda-${{ hashFiles('**/requirements*.txt') }}-exe
 
-            - name: Set up conda
+            - name: Setup conda environment
               uses: goanpeca/setup-miniconda@v1.1.2
               with:
-                  activate-environment: mac-env  # This is also hardcoded in Pyinstaller's invest.spec
+                  activate-environment: macbin-env  # This is also hardcoded in Pyinstaller's invest.spec

```

Comment:
Same comment as above, I think this code comment may be outdated.

Reply:
Removed!